presto-product-tests/conf/tempto/tempto-configuration-for-docker-ldap.yaml

```diff
@@ -4,9 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
-    cli_ldap_authentication: true
     cli_ldap_truststore_path: /etc/ldap/cacerts.jks
     cli_ldap_truststore_password: testldap
-    cli_ldap_user_name: ParentGroupUser
+    cli_ldap_user_name: DefaultGroupUser
```

Comment:
Yes. I won't merge until Sanjay's PR gets merged. So this will be eventually removed when resolving merge conflicts.